ui/admin/app/routes/scopes/scope/targets/index.js

```diff
@@ -60,20 +60,21 @@ export default class ScopesScopeTargetsIndexRoute extends Route {
       filters.type.push({ equals: type });
     });
 
-    const sessions = await this.store.query('session', {
-      query: {
-        filters: {
-          scope_id: [{ equals: scope_id }],
-          status: [
-            { equals: STATUS_SESSION_ACTIVE },
-            { equals: STATUS_SESSION_PENDING },
-          ],
+    let sessions = [];
```

Comment:
small nitpick: is it necessary to initialize sessions outside the if statement if its only used to be passed into the addActiveSessionFilters function